src/main/resources/content/k8s/avoid-running-privileged-containers-.json

```diff
@@ -1 +1,8 @@
-{"severity":"high","rule":"$.spec.template.spec.containers[*].securityContext.privileged is true","id":"92714c07-d12b-4635-ae6a-514c5c428c5a","enabled":true,"resourceType":"k8s","policy":"avoid running privileged containers "}
\ No newline at end of file
+{
+  "severity": "high",
+  "rule": "$.spec.template.spec.containers[*].securityContext.privileged is true",
+  "id": "92714c07-d12b-4635-ae6a-514c5c428c5a",
+  "enabled": true,
+  "resourceType": "k8s",
+  "policy": "avoid running privileged containers "
+}
```

Comment:
description field missing here as well.
Also, the policy name is not sentence case.